language/move-prover/bytecode/src/access_path_trie.rs

```diff
@@ -472,6 +520,21 @@ impl<T: FootprintDomain> AccessPathTrie<T> {
         })
     }
 
+    /// Apply `f` to each (access path, data) pair encoded in `self`
+    /// and collects the result when `f` returns `Some(r)`
+    pub fn map_paths<F, R>(&self, mut f: F) -> Vec<R>
```

Comment:
Nice, this looks useful!

Reply:
Actually, given the signature I was thinking that `filter_map_paths` might be a better name as the behavior is closer to https://doc.rust-lang.org/std/iter/trait.Iterator.html#method.filter_map than to https://doc.rust-lang.org/std/iter/trait.Iterator.html#method.map
What do you think?